central/productusage/service/singleton.go

```diff
@@ -0,0 +1,22 @@
+package usage
+
+import (
+	datastore "github.com/stackrox/rox/central/productusage/datastore/securedunits"
+	"github.com/stackrox/rox/pkg/logging"
+	"github.com/stackrox/rox/pkg/sync"
+)
+
+var (
+	svc  Service
+	once sync.Once
+
+	log = logging.LoggerForModule()
```

Comment:
looks unused

Reply:
yeah, used to be used. May be used in later PRs.